dss/api/collections.py

```diff
@@ -46,6 +46,24 @@ def get_impl(uuid: str, replica: str, version: str = None):
         raise DSSException(404, "not_found", "Could not find collection for UUID {}".format(uuid))
     return json.loads(collection_blob)
 
+@dss_handler
+@security.authorized_group_required(['hca'])
+def listc(replica: str):
```

Comment:
Due to the manner in which collections are assembled in the DSS, this list is likely to be long. We should consider adding support for paging relatively soon.

Reply:
I completely agree.  Should have a PR up pretty soon after this is merged.